src/components/MainView.js

```diff
@@ -85,9 +87,7 @@ function MainView({ children }) {
               flex-grow: 1;
               height: 100%;
               position: relative;
-              ${connectedGarden && !mobileMode
-                ? `margin-left: ${9 * GU}px;`
-                : ''}
+              ${hasSidebar && !mobileMode ? `margin-left: ${9 * GU}px;` : ''}
```

Comment:
This a more general fix as the margin was not being applied when we are in the profile page for example